src/perf/lib/PerfServer.cpp

```diff
@@ -71,6 +84,39 @@ PerfServer::Init(
         }
     }
 
+    if (TryGetVariableUnitValue(argc, argv, "delay", &DelayMicroseconds, nullptr)) {
+        const char* DelayTypeString = nullptr;
+        DelayType = SYNTHETIC_DELAY_FIXED;
+
+        if (TryGetValue(argc, argv, "delayType", &DelayTypeString)) {
+#ifndef _KERNEL_MODE
+            if (IsValue(DelayTypeString, "variable")) {
+                //
+                // Initialize variable delay parameters
```

Comment:
IMO, this comment isn't strictly necessary. It's fairly clear _what_ you're doing. I personally feel comments are useful if there is a _why_ or if the _what_ is really complex.